pkg/scene/widget_zone_test.go

```diff
@@ -0,0 +1,84 @@
+package scene
+
+import (
+	"testing"
+
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestWidgetZone_Find(t *testing.T) {
+	wid := id.NewWidgetID()
+
+	was := NewWidgetAlignSystem()
```

Comment:
```suggestion
	zone := NewWidgetZone()
```

You cannot use WidgetAlignSystem in `TestWidgetZone_*`